docs/pages-for-subheaders/cis-scans.md

```diff
@@ -30,37 +29,48 @@ Some tests are designated as "Not Applicable." These tests will not be run on an
 
 The report contains the following information:
 
-| Column in Report | Description |
-|------------------|-------------|
-| `id`               | The ID number of the CIS Benchmark. |
-| `description`      | The description of the CIS Benchmark test. |
-| `remediation`      | What needs to be fixed in order to pass the test. |
-| `state`    | Indicates if the test passed, failed, was skipped, or was not applicable. |
-| `node_type`        | The node role, which affects which tests are run on the node. Master tests are run on controlplane nodes, etcd tests are run on etcd nodes, and node tests are run on the worker nodes. |
-| `audit` | This is the audit check that `kube-bench` runs for this test. |
-| `audit_config` | Any configuration applicable to the audit script. |
-| `test_info` | Test-related info as reported by `kube-bench`, if any. |
-| `commands` | Test-related commands as reported by `kube-bench`, if any. |
-| `config_commands` | Test-related configuration data as reported by `kube-bench`, if any. |
-| `actual_value` | The test's actual value, present if reported by `kube-bench`. |
-| `expected_result` | The test's expected result, present if reported by `kube-bench`. |
+| Column in Report  | Description                                                                                                                                                                             |
+|-------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `id`              | The ID number of the CIS Benchmark.                                                                                                                                                     |
+| `description`     | The description of the CIS Benchmark test.                                                                                                                                              |
+| `remediation`     | What needs to be fixed in order to pass the test.                                                                                                                                       |
+| `state`           | Indicates if the test passed, failed, was skipped, or was not applicable.                                                                                                               |
+| `node_type`       | The node role, which affects which tests are run on the node. Master tests are run on controlplane nodes, etcd tests are run on etcd nodes, and node tests are run on the worker nodes. |
```

Comment:
Should it be controlplane or control plane?

Reply:
I find the usage of both in multiple places across the documentation, so I am not sure. 
@btat Do you have any suggestions?

Reply:
We should go with control plane since that's what the K8s docs use. I'll fix the inconsistent usage in our docs.

Reply:
Cool, if so I will not change it here but leave it to you.